test/unit/node-labels.test.js

```diff
@@ -658,6 +658,28 @@ tap.test('label: "build" when ./android-configure has been changed', (t) => {
 
       t.same(resolved, labels)
 
+      t.end()
+    })
+  }
+});
+
+[
+  [ ['worker'],
```

Comment:
Does this catch changes to [`lib/worker_threads.js`](https://github.com/nodejs/node/blob/master/lib/worker_threads.js)?

Reply:
Added now.

I forgot this before😅.